release-notes/unreleased/IGAPP-839-Fix-Paragraph-Overflow-Native.yml

```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-839
+show_in_stores: true
+platforms:
+  - ios
+  - android
+en: Fixed faulty display of paragraphs in webview.
+de: Fehlerhafte Darstellung von Paragraphen im Webview wurde behoben.
```

Comment:
These are implementation details that rather confuse users I think. Better to just use something generic like ` Fixed faulty display in certain cases.` or simlar